stl/src/vector_algorithms.cpp

```diff
@@ -2192,6 +2215,38 @@ extern "C" {
 __declspec(noalias) void __stdcall __std_bitset_to_string_1(
     char* const _Dest, const void* _Src, size_t _Size_bits, const char _Elem0, const char _Elem1) noexcept {
 #ifndef _M_ARM64EC
+    if (_Use_avx2() && _Size_bits >= 256) {

```

Comment:
`VSO_0000000_vector_algorithms` should be extended with correctness coverage - right now we max out at 75 bits. We already have the logic to test AVX2 enabled and disabled.

Reply:
There's randomized test up to 2048 bits below.
I'm not sure if it is needed to test with literals as well.